api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -96,10 +99,19 @@ message DownstreamTlsContext {
 }
 
 // TLS context shared by both client and server TLS contexts.
-// [#next-free-field: 9]
+// [#next-free-field: 11]
 message CommonTlsContext {
   option (udpa.annotations.versioning).previous_message_type = "envoy.api.v2.auth.CommonTlsContext";
 
+  // Config for CA client plugin to get secrets
```

Comment:
FWIW, maybe we should drop the terminology "plugin". This is inconsistent with the rest of the Envoy API (except Wasm, making it even more confusing). I think these are just CAClientExtensions or something like that.

Reply:
I'll replace "plugin" with "extension" everywhere. Is that okay?

Reply:
Sure